rct229/ruleset_functions/compare_standard_val.py

```diff
@@ -0,0 +1,31 @@
+from rct229.utils.assertions import assert_
+from rct229.utils.std_comparisons import std_equal
+
```

Comment:
Isn't ```import operator``` required because ```operator(val, std_val)``` part in line 29? I also see ```import operator``` in ```rct229/ruleset_functions/compare_standard_val_test.py``` file. I'd appreciate it if you could double-check this.

Reply:
I think the `operator` will need to be imported once I updated code following my previous comments. I kind of like the way it was implemented to avoid importing operators in the rules. @charliepnnl @yunjoonjung-PNNL Let me know if you think the same way. I can update the function.